watch.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+vagrant ssh -c "python3 /usr/local/submitty/GIT_CHECKOUT/Submitty/.setup/bin/code_watcher.py"
```

Comment:
This is cool and all but it has nothing to do with the issue you are solving. :/ Make separate PR for it?